src/utils/data_loaders.py

```diff
@@ -0,0 +1,190 @@
+"""
+Functions having to do with loading data from output of
+files downloaded in scripts/download_data_glue.py
+
+"""
+from .tokenizers import OpenAIBPETokenizer, MosesTokenizer, AVAILABLE_TOKENIZERS
```

Comment:
Do we need to import the `OpenAIBPETokenizer` and `MosesTokenizer` symbols anymore?